.github/workflows/cleanup_caches.yml

```diff
@@ -0,0 +1,36 @@
+name: Cleanup caches
+on:
+  workflow_dispatch:
+  schedule:
+    # at 00:00 on workdays
+    - cron: '0 0 * * 1,2,3,4,5'
+
+permissions: read-all
+
+jobs:
+  Cleanup_OV_CACHE:
+    name: Cleanup OV_CACHE
+    runs-on: aks-linux-medium
+    if: ${{ github.repository_owner == 'openvinotoolkit' }}
+    container:
+      image: openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04
+      volumes:
+        - /mount:/mount
+    env:
+      OV_CACHE: /mount/caches/huggingface/ov_cache
+
+    steps:
+      - name: Pre-Collecting Cache Info
+        run: |
+          echo "Cache info: "
+          du -h -d2 ${{ env.OV_CACHE }}
+
+      - name: Cleanup cache
+        run: |
+          echo "Delete cache files if they have not been used in over 30 days"
```

Comment:
Is it really supposed to cleanup cache files not used over 30 days? I see that it just wipes out the whole directory once a day (probably not very safe to do in runtime).

Reply:
yes, because we need to convert the models any time when the OV runtime is updated, so the easiest way to remove the once per day